### PR TITLE
Create artipie setting storage

### LINKED ISSUE #3: Parse artipie configuration file
Parse Artipie YAML configuration file from `/etc/artipie/artipie.yml` and create `Storage` instance based on these files (maybe create `BlockingStorage` instead of `Storage`).

### PR DESCRIPTION
**Pull request by olenagerasimova**
Closes #3 
Added service start argument to accept path to config file and implemented artipie configuration storage creation from it.

### CODE REVIEW

File: src/main/java/com/artipie/front/Service.java [hunk 1]
```diff
@@ -52,26 +64,33 @@ private Service(final Storage storage) {
...
+                    // @checkstyle LineLengthCheck (1 line)
+                    "Set path to artipie yaml configuration file with the `f` or `config-file` argument"
+                );
+            }
         } catch (final ParseException ex) {
             final HelpFormatter formatter = new HelpFormatter();
             formatter.printHelp("com.artipie.front.Service", options);
             throw ex;
+        } catch (final IOException ex) {
+            Logger.error(Service.class, "Failed to read artipie setting yaml");
```
Comment (kirill-h5i) [inline src/main/java/com/artipie/front/Service.java:92]: I think we need to propagate exception to exit process with non-zero code

File: src/main/java/com/artipie/front/Service.java [hunk 2]
```diff
@@ -52,26 +64,33 @@ private Service(final Storage storage) {
...
-            true,
-            "service port. Should be int value. Default value 8080"
-        );
+        options.addOption(Service.PORT);
+        options.addOption(Service.CONFIG);
         final CommandLineParser parser = new DefaultParser();
         final CommandLine cmd;
         try {
             cmd = parser.parse(options, args);
+            if (cmd.hasOption(Service.CONFIG)) {
```
Comment (kirill-h5i) [inline src/main/java/com/artipie/front/Service.java:73]: We don't need this manual check if we set `CONFIG` option as `required: true`

**Review by kirill-h5i — CHANGES_REQUESTED** [review]
@olenagerasimova please check my comments

**Comment by olenagerasimova** [comment]
@kirill-h5i thanks, corrected

**Review by g4s8 — APPROVED** [review]

**Pull request merged**